docs/spec.bs

```diff
@@ -49,36 +52,45 @@ Markup Shorthands: markdown yes, css no
 
 The <dfn for="web environment">web environment</dfn> is defined as <i>TODO</i>
 
-# Attesters # {#attester}
+The <dfn for="content binding">content binding</dfn> is defined as <i>TODO</i>
 
-The term <dfn for="attester">attester</dfn> refers to a third party capable of returning an [=Integrity verdict=]. A <dfn for="Integrity verdict">Integrity verdict</dfn> refers
-to a response that confirms if the [=attester=] trusts the [=web environment=] the [=user agent=] is
-executing in.
+# Infrastructure # {#infrastructure}
 
-The [=user agent=] connects to the [=attester=] through an {{AttesterConnection}}.
+## Attesters ## {#infrastructure-attester}
 
-The [=user agent=] SHOULD use separate {{AttesterConnection}}s if the {{AttesterConnection}}
-stores state in the [=Integrity verdict=] that can be used for cross site tracking.
+The term <dfn for="attester">attester</dfn> refers to a third party capable of returning an
+[=attester verdict=].
 
-## Token Format ## {#attester-tokens}
+[=User agents=] MUST have an [=attester connection=] to an [=attester=] in order to use Web
+Environment Integrity.
 
-<i>Todo</i>
+The [=user agent=] SHOULD use separate [=attester connections=] if the [=attester connection=]
+stores state in the [=attester verdict=] that can be used for cross site tracking.
 
-## Browser Acceptance Requirements ## {#attester-browser-requirements}
+### [=Attester Connection=] ### {#attester-connection}
 
-<i>Todo</i>
+The <dfn for="Attester Connection">attester connection</dfn> is an abstract concept representing the
+channel through which the [=user agent=] can communicate to an [=attester=].
+The [=user agent=] uses the [=attester connection=] to request new [=attester verdicts=].
 
-## {{AttesterConnection}} ## {#attester-connection}
+The [=attester connection=] MUST use a [=content binding=] to create a new [=attester verdict=].
```

Comment:
I think another way of phrasing that would be that the attester MUST always send a refreshed attester verdict.

However, if the domain and the CB are the same, I think it would be reasonable for the attester to use a cached verdict depending on how they calculate it. I'll hold off adding this for the moment but we can discuss this detail more if you want to chat about it more and possibly follow up.